app/main.py

```diff
@@ -92,6 +107,21 @@
 # ----------------------------------------------------------------------------------------------------------------------
 # General callbacks
 
+@app.callback(
+    Output("page-content", "children"),
+    Input("url", "pathname")
+)
+def display_page(pathname):
+    """
+    This callback takes the url path as input and returns the corresponding page layout,
+    thanks to the instantiation functions built in the various .py files.
+    """
+    if pathname == "/alert_screen":
+        return alert_screen()
+    else:
+        return Homepage()
```

Comment:
@pechouc we used to have this `display_page` callback, but I do not remember where it came and by what is has been replaced ?

Reply:
It is mean to have multiple pages app. Here it means that `http://127.0.0.1:8050/alert_screen` will point to the `alert_screen()` function. Whereas `http://127.0.0.1:8050/` or `http://127.0.0.1:8050/whatever_you_want` will point to `homepage()`function

Reply:
I know and we used to have this multiple pages (/alerts, /risks) display page function. I was just wondering why we do not have it anymore but now I remember that deleted when we moved to a 'one pager' platform format.

Reply:
This was from this [beloved tutorial](https://towardsdatascience.com/create-a-multipage-dash-application-eceac464de91?gi=8a94d5ddbb31) but yes, we killed at some point during the DfG season.